portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,207 @@
+        <p>The robot motors are currently powered with 4 x AA Batteries and the Raspberry Pi is powered by a 5V Power Bank.</p>
+
+        <p>Software</p>
+        <p>We forked a project from a blog which created a mechanical hand which can draw out SVG images.</p>
+        <p>So, we inherited the methods for parsing the SVG file and for performing interpolation on the Bezier curves found inside it to obtain the pixels.</p>
+        <p>However, we had to convert the pixels generated from the SVG into vectors for drawing, and further process it into relative angle and magnitude for the robot to move to for each stroke.</p>
+        <p>Finally, a separate program had to be written to read the angles and magnitudes generated and process it to be sent as signals for activating and deactivating the bot's motors.</p>
+
+    </div>
+
+    <hr class="dashed">
+
+    <div class="container" id="contact">
+    <form action="action_page.php">
+
+    <label for="fname">First Name</label>
+    <input type="text" id="fname" name="firstname" placeholder="Your name..">
+
+    <label for="lname">Last Name</label>
+    <input type="text" id="lname" name="lastname" placeholder="Your last name..">
+
+    <label for="country">Country</label>
+    <select id="country" name="country">
+      <option value="singapore">Singapore</option>
+      <option value="usa">USA</option>
+      <option value="others">Others</option>
+    </select>
+
+    <label for="subject">Subject</label>
+    <textarea id="subject" name="subject" placeholder="Write something.." style="height:200px"></textarea>
+
+    <input type="submit" value="Submit">
+
+  </form>
+</div>
+
+    <!-- Image Directory
+    <section class="w3-row-padding w3-center w3-light-grey">
+    <article class="w3-third">
+        <p>Resume</p>
+        <img src="images/resume.png" alt="Random Name" style="height:500px">
+        <p>Look at my Experiences</p>
+    </article>
+    <article class="w3-third">
+        <p>Projects</p>
+        <img src="images/projects.png" alt="Random Name" style="height:500px">
+        <p>Check out my creations!</p>
+    </article>
+    <article class="w3-third">
+        <p>Contact</p>
+        <img src="images/contact_me.jpg" alt="Random Name" style="height:500px">
+        <p>Contact me</p>
+    </article>
+    </section> -->
+
+
+    <!-- Footer -->
+    <footer class="w3-container w3-padding-64 w3-center w3-black w3-xlarge">
+        <a href="https://github.com/leafgecko"><i class="fa fa-github"></i></a>
+        <!-- <a href="#"><i class="fa fa-twitter"></i></a> -->
+        <a href="https://www.linkedin.com/in/katherine-kee-wan-ting/"><i class="fa fa-linkedin"></i></a>
+        <p class="w3-medium">
+        Created by Katherine Kee Wan Ting </p>
+    </footer>
+
+    <script>
+    // Automatic Slideshow - change image every 3 seconds
+    var myIndex = 0;
```

Comment:
Please move js to script.js, or a new js file.

E.g, `<script>showCarousel()</script>`

Reply:
moved to script.js! :)